add cross compile tutorial

etrombly opened the pull request:
Added a tutorial for cross compiling from Linux to Windows

GuillaumeGomez commented:
Thanks so much about this!

@EPashkin: Seems ok for you? (I'll add the missing footer links once it's merged)

EPashkin commented on tuto/cross.md:
```diff
@@ -0,0 +1,41 @@
+The package installation steps are for arch, but should be similar for other distros.
```
"Arch Linux" instead "arch" ?

EPashkin commented on tuto/cross.md:
```diff
@@ -0,0 +1,41 @@
+
+Now install mingw and gtk libraries, this will take a while
+
+    pacaur -S mingw-w64-gcc mingw-w64-freetype2-bootstrap mingw-w64-cairo-bootstrap
```
Maybe change it to `pacman`? Or default package manager changed?

etrombly replied:
mingw is in the AUR, so it won't work with pacman. I can try it in ubuntu in a bit so we can have instructions for both if you'd like.

EPashkin replied:
Understand now.
Can you add comment about this, if AUR don't installed by default?

EPashkin commented:
@etrombly Thanks, now we will have documented answer to "How distribute gtk application?" 😉

etrombly commented:
Updated the formatting a little, made the Arch section more clear. Also added instructions to add a program icon and change the theme to a windows 10 native look. If you want you should be able to adapt those sections pretty easily for someone building on windows, I haven't tried. Didn't end up adding Ubuntu instructions.

GuillaumeGomez commented:
Thanks a lot! (you even added footer urls, awesome!)
